pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -104,12 +105,17 @@ func NewReconciler() *Reconciler {
 	}
 }
 
-func generateTenantRulesData(policyName string, protoKeywords map[string]sets.String) *bytes.Buffer {
+func generateTenantRulesData(policyName string, protoKeywords map[string]sets.String, enableLogging bool) *bytes.Buffer {
 	rulesData := bytes.NewBuffer(nil)
 	sid := 1
 
+	var tagKeyword string
+	if enableLogging {
+		tagKeyword = " tag: session, 30, seconds;"
```

Comment:
I didn't find doc about these keywords, could you share a link and add comment to help understand?

Reply:
Suricata lacks the documentation for "tag", which is an [open ticket](https://redmine.openinfosecfoundation.org/issues/3015). Looking into the codebase, it's handled in their file `detect-engine-tag`. Also Snort has a similar functionality, [doc](https://www.snort.org/faq/readme-tag).

Reply:
could you add this information to code comment? otherwise people who looks at this will have the same question.